### PR TITLE
[bitnami/kafka] Kafka-Exporter can start with read only rootfs

### LINKED ISSUE #5931: [btinami/kafka] Kafka-Exporter doesn't start with read only rootfs
**Which chart**:
kafka 12.13.2 

**Describe the bug**
Kafka-Exporter Container dies with following log msg:
`bash: cannot create temp file for here-document: Read-only file system`

**To Reproduce**
create a PSP with norootfs and assign to default serviceuser

**Expected behavior**
Container can start with read-only root filesystem 

**Version of Helm and Kubernetes**:

- Output of `helm version`:

```
version.BuildInfo{Version:"v3.4.2", GitCommit:"23dd3af5e19a02d4f4baa5b2f242645a1a3af629", GitTreeState:"clean", GoVersion:"go1.14.13"}
```

- Output of `kubectl version`:

```
Client Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.0", GitCommit:"9e991415386e4cf155a24b1da15becaa390438d8", GitTreeState:"clean", BuildDate:"2020-03-25T14:58:59Z", GoVersion:"go1.13.8", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.6", GitCommit:"fbf646b339dc52336b55d8ec85c181981b86331a", GitTreeState:"clean", BuildDate:"2020-12-18T12:01:36Z", GoVersion:"go1.15.5", Compiler:"gc", Platform:"linux/amd64"}

```

The problem is here 
https://github.com/bitnami/charts/blob/4a55a6f18c53ada9b87db23819ed2df7527dddd2/bitnami/kafka/templates/kafka-metrics-deployment.yaml#L41

I start the container with `/bin/bash -c sleep infinity`
`I have no name!@kafka-exporter-647c66c687-lstrf:/opt/bitnami/kafka-exporter$ read -r -a sasl_passwords <<< "$(tr ',;' ' ' <<< "${SASL_USER_PASSWORD}")"
bash: cannot create temp file for here-document: Read-only file system
`

### PR DESCRIPTION
**Pull request by sveba**
**Description of the change**

Helps starting kafka-exporter in read-only mounted filesystem

**Benefits**

no need for tmp-files

**Possible drawbacks**

none

**Applicable issues**

  - fixes #5931

**Additional information**

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)

### CODE REVIEW

File: bitnami/kafka/Chart.yaml [hunk 1]
```diff
@@ -29,4 +29,4 @@ name: kafka
 sources:
   - https://github.com/bitnami/bitnami-docker-kafka
   - https://kafka.apache.org/
-version: 12.13.3
+version: 12.13.6
```
Comment (carrodher) [inline bitnami/kafka/Chart.yaml:32]: There is a typo bumping the chart version
```suggestion
version: 12.13.4
```

**Comment by sveba** [comment]
any comment on this? we need this rather quick ;)

**Review by marcosbc — APPROVED** [review]
LGTM, thanks for the contribution!

**Comment by marcosbc** [comment]
@sveba Our internal pipeline will now detect this change and publish the new Helm chart version. Please mind that it may take a few hours though.

**Pull request merged**